File: 0242_Valid_Anagram/242.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        return sorted(s) == sorted(t)

a = Solution()
print(a.isAnagram(s = "anagram", t = "nagaram"))
print(a.isAnagram(s = "rat", t = "car"))

File: 0242_Valid_Anagram/242.cpp
#include <bits/stdc++.h>
using namespace std;
class Solution
{
public:
    bool isAnagram(string s, string t)
    {
        sort(s.begin(), s.end());
        sort(t.begin(), t.end());
        return s == t;
    }
};
int main()
{
    Solution a;
    cout << a.isAnagram("anagram", "nagaram") << "\n"
         << a.isAnagram("rat", "car");
    return 0;
}